Aptoide/aptoide-client-v8
# Bugfix/asv 1146 donations metrics

**What does this PR do?**

This PR aims to implement metrics for the donations flow. The implemented events can be found in the ticket;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] DonationsAnalytics.java

**How should this be manually tested?**

Do the actions that correspond to the events and use charles to verify they're being correctly sent;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1146](<https://aptoide.atlassian.net/browse/ASV-1146>)

**Code Review Checklist**
